providers/axfrddns/axfrddnsProvider.go

```diff
@@ -52,32 +50,39 @@ var features = providers.DocumentationNotes{
 	providers.CanUseSSHFP:            providers.Can(),
 	providers.CanUseSVCB:             providers.Can(),
 	providers.CanUseTLSA:             providers.Can(),
-	providers.DocCreateDomains:       providers.Cannot(),
 	providers.DocDualHost:            providers.Cannot(),
 	providers.DocOfficiallySupported: providers.Cannot(),
+	// Possible to support via catalog zones (RFC 9432), but those are not
+	// directly supported by DNSControl right now (although nothing is stopping
+	// you from manually updating a catalog zone using DNSControl if you wish).
+	providers.CanGetZones:      providers.Cannot(),
```

Comment:
What's the meaning of  `CanGetZones` ? If it means that the `get-zone` command is available, then, it is available with `AXFR`, isn't it ?

Reply:
CanGetZones  means that the provider is able to get a list of zones at the server, and therefore know if a zone doesn't exist and needs to be created.

Each provider tends to have a bespoke way to do that. Since AXFRDDNS is generic and works with many providers, I think it is best to leave that as Cannot().  (Though, I'd be glad to be wrong!)